socorro/external/postgresql/staticdata.py

```diff
@@ -189,8 +212,79 @@ class ReportPartitionInfo(BaseTable):
              'date_processed', 'DATE']]
 
 
+class Products(BaseTable):
+    table = 'products'
+    columns = [
+        'product_name', 'sort', 'rapid_release_version', 'release_name', 'rapid_beta_version'
+    ]
+    rows = [
+        ['Fennec', '3', '5.0', 'mobile', '999.0'],
+        ['Thunderbird', '2', '6.0', 'thunderbird', '999.0'],
+        ['Firefox', '1', '5.0', 'firefox', '23.0'],
+        ['SeaMonkey', '6', '2.3', 'seamonkey', '999.0'],
+        ['FennecAndroid', '4', '5.0', '**SPECIAL**', '999.0'],
+    ]
+
+
+class ProductBuildTypes(BaseTable):
+    table = 'product_build_types'
+    columns = [
+        'product_name', 'build_type'
+    ]
+    rows = [
+        ['Firefox', 'release'],
```

Comment:
Make this `['Firefox', 'release', 0.1],` and for all other 2-item lists make it `1.0` instead of `0.1`. E.g. `['Firefox', 'beta', 1.0]`.